test/wrappers/vroom_test.rb

```diff
@@ -425,7 +424,6 @@ def test_with_rest_at_the_end
           end: 20000
         },
         rest_ids: ['rest_a'],
-        cost_late_multiplier: 1
```

Comment:
Let me guess, we weren't calling Vroom because of these costs, is that it?

Reply:
Vroom is still called even with cost_late_multiplier, but, as we remove the timewindow.end from the vehicle, the rest is placed in first position of the route